src/app/core/services/mocks/mock.github.service.ts

```diff
@@ -0,0 +1,390 @@
+import { getNumberOfPages } from '../../../shared/lib/github-paginator-parser';
+import { IssueComment } from '../../models/comment.model';
+import { ErrorHandlingService } from '../error-handling.service';
+import { GithubUser } from '../../models/github-user.model';
+import { GithubIssue } from '../../models/github/github-issue.model';
+import { GithubComment } from '../../models/github/github-comment.model';
+import { GithubResponse } from '../../models/github/github-response.model';
+import { IssuesCacheManager } from '../../models/github/cache-manager/issues-cache-manager.model';
+import { IssueLastModifiedManagerModel } from '../../models/github/cache-manager/issue-last-modified-manager.model';
+import { Apollo, QueryRef } from 'apollo-angular';
+import {
+  FetchIssue,
+  FetchIssueQuery, FetchIssues, FetchIssuesByTeam, FetchIssuesByTeamQuery, FetchIssuesQuery,
+} from '../../../../../graphql/graphql-types';
+import { GithubGraphqlIssue } from '../../models/github/github-graphql.issue';
+import { ApolloQueryResult } from 'apollo-client';
+import { HttpErrorResponse } from '@angular/common/http';
+import RestGithubIssueFilter from '../../models/github/github-issue-filter.model';
+import { DocumentNode } from 'graphql';
+import { ElectronService } from '../electron.service';
+import { AppConfig } from '../../../../environments/environment.test';
+import { Phase } from '../phase.service';
+import { SessionData } from '../../models/session.model';
+import { GithubRelease } from '../../models/github/github.release';
+import { LabelService } from '../label.service';
+import { Label } from '../../models/label.model';
+
+const Octokit = require('@octokit/rest');
+const CATCHER_ORG = 'CATcher-org';
+const CATCHER_REPO = 'CATcher';
+
+let ORG_NAME = '';
+let MOD_ORG = '';
+let REPO = '';
+let DATA_REPO = '';
+let octokit = new Octokit();
+
+@Injectable()
+export class MockGithubService {
+  private static readonly IF_NONE_MATCH_EMPTY = { 'If-None-Match': '' };
+
+  private issuesCacheManager = new IssuesCacheManager();
+  private issuesLastModifiedManager = new IssueLastModifiedManagerModel();
+  private issueQueryRefs = new Map<Number, QueryRef<FetchIssueQuery>>();
+
+  constructor(
+    private errorHandlingService: ErrorHandlingService,
+    private apollo: Apollo,
+    private electronService: ElectronService,
+  ) {}
+
+  storeOAuthAccessToken(accessToken: string) {
+    octokit = new Octokit({
+      auth() {
+        return `Token ${accessToken}`;
+      }
+    });
+  }
+
+  storeOrganizationDetails(orgName: string, dataRepo: string) {
+    MOD_ORG = orgName;
+    DATA_REPO = dataRepo;
+  }
+
+  storePhaseDetails(phaseRepoOwner: string, repoName: string) {
+    REPO = repoName;
+    ORG_NAME = phaseRepoOwner;
+  }
+
+  fetchIssuesGraphqlByTeam(tutorial: string, team: string, issuesFilter: RestGithubIssueFilter): Observable<Array<GithubIssue>> {
+    const graphqlFilter = issuesFilter.convertToGraphqlFilter();
```

Comment:
In the mock GitHubService, can we remove the logic used to fetch issues via GraphQL ? (since it won't be used).

Reply:
Yes, I'm currently removing code snippets and making sure that it does not break.

Given that we are not talking to the GH backend at all, do you think removing other parts, like labelCreation, in the github service be okay?